flask-app/app.py

```diff
@@ -25,7 +25,7 @@
 def retrieveUnlabelledClips(count):
   query = """ SELECT url, start, end FROM labelled
               WHERE completed = FALSE
-              ORDER BY RAND()
+              ORDER BY priority ASC, RAND()
```

Comment:
Oh - for some reason I was expecting some next level stochastic randomizing ting. Quick n slick! LGTM <3